kr-core/src/main/clojure/edu/ucdenver/ccp/kr/sparql.clj

```diff
@@ -430,7 +474,11 @@
         non-vars (symbols-no-vars triple-pattern)
         namespaces (distinct (map namespace non-vars))
         prefixes (get-prefixes-from-namespaces namespaces)]
-    (str
+    (println "options" options)
+    (println "non-vars" non-vars)
+    (println "prefixes" prefixes)
+    (println "namespaces" namespaces)
```

Comment:
Are these `println` statements will necessary, or were they added for debugging?  If they're generally useful, we should start using [logging](https://github.com/clojure/tools.logging) to report these so that we have more control over the output.